great_expectations/checkpoint/actions.py

```diff
@@ -179,42 +200,35 @@ class SlackNotificationAction(DataDocsAction):
         show_failed_expectations: Shows a list of failed expectation types.
     """
 
-    def __init__(  # noqa: PLR0913
-        self,
-        renderer: dict,
-        slack_webhook: Optional[str] = None,
-        slack_token: Optional[str] = None,
-        slack_channel: Optional[str] = None,
-        notify_on: str = "all",
-        notify_with: Optional[list[str]] = None,
-        show_failed_expectations: bool = False,
-    ) -> None:
-        """Create a SlackNotificationAction"""
-        super().__init__()
-        self.renderer = instantiate_class_from_config(
-            config=renderer,
-            runtime_environment={},
-            config_defaults={},
-        )
-        module_name = renderer["module_name"]
-        if not self.renderer:
-            raise ClassInstantiationError(
-                module_name=module_name,
-                package_name=None,
-                class_name=renderer["class_name"],
+    slack_webhook: Optional[str] = None
+    slack_token: Optional[str] = None
+    slack_channel: Optional[str] = None
+    notify_on: Literal["all", "failure", "success"] = "all"
+    notify_with: Optional[list[str]] = None
+    show_failed_expectations: bool = False
+    renderer: dict = {
+        "class_name": "SlackRenderer",
+        "module_name": "great_expectations.render.renderer.slack_renderer",
+    }
```

Comment:
Note that I've added default values here. I'd imagine this covers 99% of usecases